telemetry/telemetry.mk

```diff
@@ -0,0 +1,16 @@
+################################################################################
+#
+# telemetry
+#
+################################################################################
+
+TELEMETRY_VERSION = v0.91
+TELEMETRY_SITE = $(call github,Outernet-Project,telemetry_client,$(TELEMETRY_VERSION))
+TELEMETRY_SETUP_TYPE = setuptools
+
+define TELEMETRY_INSTALL_INIT_SYSV
+	$(INSTALL) -Dm0755 $(BR2_EXTERNAL)/package/telemetry/S91telemetry \
```

Comment:
`$(BR2_EXTERNAL)/package/telemetry` can be replaced with `$(call epkgdir,telemetry)`